.jazzy.yaml

```diff
@@ -1,7 +1,6 @@
 author: Hyperwallet Systems Inc
 clean: true
 author_url: https://www.hyperwallet.com/
-github_file_prefix: https://github.com/hyperwallet/hyperwallet-ios-core-sdk-sandbox
 github_url: https://github.com/hyperwallet/hyperwallet-ios-core-sdk-sandbox
```

Comment:
incorrect url - Correct one is `https://github.com/hyperwallet/hyperwallet-ios-sdk`

Reply:
Changed the url